roles/StackStorm.nodejs/tasks/nodejs_yum.yml

```
@@ -10,6 +10,22 @@
   until: _task is succeeded
   tags: nodejs
 
+# nodejs updates the version in their repo rpm without updating the version of the rpm itself.
+- name: Determine if nodesource repo upgrade is required
```

```suggestion
- name: Determine if nodejs repo is out of date (happens when upgrading nodejs to a new major version)
```
Does this make it more clear?